setup.py

```diff
@@ -163,4 +166,10 @@ def run(self):
 
 
 if __name__=="__main__":
+
+    example_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                'panel','examples')
+    if 'develop' not in sys.argv:
```

Comment:
Don't really get this, develop installs the project in place which means you're basically just copying the files to the same location.

Reply:
Sorry my mistake, missed the **not**.